frontends: added css changes for dark mode

thisconnect opened the pull request:
Changed existing CSS and assets to hardcoded dark mode, the related JavaScript will be done in a later commit. Some assets have not yet been updated as they require some additional logic to change to dark mode.

superseds https://github.com/digitalbitbox/bitbox-wallet-app/pull/1883

shonsirsha reviewed (approved):
looks like we missed this?

<img width="1440" alt="Screenshot 2023-01-16 at 06 08 25" src="https://user-images.githubusercontent.com/28676406/212602906-4ca19f7d-dd0c-47cf-bc22-b55f6714e5a2.png">

(buy/moonpay disclaimer)

otherwise LGTM.

thisconnect commented:
fixed "buy/moonpay disclaimer", 

https://github.com/digitalbitbox/bitbox-wallet-app/compare/df3a208d6d178fa691b95e8f544f2c7a42619c14..6db716ddaaec632fbaaad489e9dbcfbb84ef3923

updated commit message as there are other things that are missing, but require some extra logic to show in light or dark mode. 

i.e. account summary loading, for example the disabled week/month buttons
    
![Screen Shot 2023-01-16 at 14 24 33](https://user-images.githubusercontent.com/546900/212690411-2a081f3b-e3bc-43ec-8483-a363a6dae909.png)
